### ISSUE TITLE
Training/Testing on custom images

### ISSUE THREAD

**Issue by Param-Raval** [IsHYuhi/ST-CGAN_Stacked_Conditional_Generative_Adversarial_Networks#5]
@IsHYuhi  Thank you for this implementation! However, I want to use the checkpoint model to test/evaluate on my own images but I'm not able to trace which images from the ISTD dataset are used in the evaluation. I get the prediction files "test_0_"* , "test_10_"* and so on after training on ISTD. But which files are these in the ./dataset folder (so that I can try replacing with my own images)? Or is there another way to test/evaluate on my own images?
Thank you!

**Comment by IsHYuhi** [comment]
Thank you for asking @Param-Raval ! I split train images which under dataset/train/ to train and validation with Hold-Out method. (I may update these code for cross-validation (e.g. using KFold from sklearn.model_selection))

So, "test_0_*", "test_10_*" and so on use images from the validation set which is split. (This is confusing, so I will fix it too.)
if you would like to check the training set and the validation set, you could use below.
```print('train_img_list['path_A']')```, ```print('train_img_list['path_B']')```

And, if you want to test your own images, you can load a checkpoint trained with the ISTD dataset and can test them.
Also, I can add the code for testing RGB images if you want to.
Please do not hesitate to ask me 😄

Thank you.

**Comment by Param-Raval** [comment]
Thank you for the response! Can you please tell me how can I test on my own images after loading the checkpoints? (I see the commented code of loading checkpoints in _train.py_, but not sure if that's relevant or what to do next if I do that).

The code for RGB images would be great and really helpful. Thank you!

**Comment by IsHYuhi** [comment]
I didn't prepare a test module. So, I will add it within several days.
If you want to implement a test module on your own, please refer to the evaluate function in train.py, line 63, or the test code which I will commit.  👍

**Comment by IsHYuhi** [comment]
Hi @Param-Raval .
I added testing module and description. If you have any other questions, please ask me👍
If you don't have, I close this issue.
